packages/dds/tree/src/feature-libraries/sequence-change-family/sequenceEditBuilder.ts

```diff
@@ -4,27 +4,210 @@
  */
 
 import { ProgressiveEditBuilder } from "../../change-family";
+import { Transposed as T } from "../../changeset";
 import { ITreeCursor } from "../../forest";
-import { AnchorSet, UpPath, Value, Delta } from "../../tree";
+import { AnchorSet, UpPath, Value, Delta, getDepth } from "../../tree";
+import { fail } from "../../util";
+import { jsonableTreeFromCursor } from "../treeTextCursor";
 import { sequenceChangeFamily } from "./sequenceChangeFamily";
 import { SequenceChangeset } from "./sequenceChangeset";
 
 export class SequenceEditBuilder extends ProgressiveEditBuilder<SequenceChangeset> {
+    private opId: number = 0;
+
     constructor(
         deltaReceiver: (delta: Delta.Root) => void,
         anchorSet: AnchorSet,
     ) {
         super(sequenceChangeFamily, deltaReceiver, anchorSet);
     }
 
-    public setValue(node: NodePath, value: Value) { }
+    public setValue(node: NodePath, value: Value) {
+        const modify: T.Modify = { type: "Modify", value: { type: "Set", value } };
```

Comment:
I'm not sure if you care, but its worth noting that in the case where value is undefined, the object you create here has an explicit value field that's undefined.

If this type needs to JSON round trip anywhere, this will cause it to be false when doing a deep equality comparison. This was annoying in JsonableTree so I explicitly delete the value fields when they are undefined when working with them.

Reply:
Thanks. Updated the code.